### DISCUSSION TITLE
Trying to get it working with ispy/agentdvr

### DISCUSSION THREAD

**Discussion by alexbodian** [mrlt8/docker-wyze-bridge#774] [Q&A] [answered]
I have two dockers running on my home NAS with Unraid. 

Previously, I have been using AgentDVR with a few other Wyze cameras using the RTSP firmware. Recently, I got a V3 Pro and attempted to add it to my current setup using the Docker bridge.

Right now it looks like everything is fine on the bridge side and I am able to see the rtsp stream when I view it through VLC but when I try to add it to AgentDVR with the following steps I get that error.

![step1](https://user-images.githubusercontent.com/14111968/231051643-7654afed-588d-45d4-b0f7-c9bee1b23135.png)
![step2](https://user-images.githubusercontent.com/14111968/231051644-c783c28c-2564-47fd-b2c2-c326229fbdae.png)
![step3](https://user-images.githubusercontent.com/14111968/231051646-aeac7d0d-bc02-4daa-8255-648bef95c934.png)
![finalError](https://user-images.githubusercontent.com/14111968/231051642-d78cf0de-259c-4410-a7da-3d4a985c3f61.png)

**Comment by mrlt8** [comment]
If they're both on the same host, it should work with the loopback address `rtsp://127.0.0.1:8554/v3pro`
It should also work with the same rtsp address you're using to access the stream over VLC. e.g., `rtsp://192.168.0.10:8554/v3pro`.

**Reply by alexbodian** [reply, marked as answer]
It ended up being something on the agentdvr side. I set the video type as mjpeg and entered the information in there instead of doing it through the network camera wizard setup.
